### DISCUSSION TITLE
Accuracy does not contribute enough to score

### DISCUSSION THREAD

**Discussion by rozukke** [ppy/osu#20457] [Q&A] [answered]
After searching for this in discussions and not finding anything, I want to bring up the current scoring system. I have not heard anything about the future intentions/steps going forward in terms of scoring, but as it stands right now the scoring in lazer is sub-par to say the least.

![scores](https://user-images.githubusercontent.com/73006620/192105055-782c565e-7517-4296-954c-26f290c3ec1f.png)

This is a concrete example of what I'm talking about, and the scores shown here demonstrate the disparity between accuracy and achieved score on a map. In the screenshot, a score with ~7.5% lower accuracy and about 3% higher total combo results in an overall higher score. Comparing the number of 'mistakes', it comes down to ~150 vs. ~30, which is a 5x difference that is discounted due to the (in my opinion) negligible combo difference.

I also noticed this in many multiplayer matches, where people with slightly higher combo would outrank others with more than 10% higher accuracy than them and slightly lower combo.

My apologies if this is currently being worked on/improved but I could not find any discussion to reference, so this is my appeal to include a higher accuracy contribution into the current lazer scoring system.

**Comment by ghost** [comment]
(The current system seems to be, at least in standard, 70% combo and 30% accuracy)

**Comment by Kaerulans** [comment, marked as answer]
This seems like a duplicate of #18260
